calico-enterprise_versioned_docs/version-3.18/releases.json

```diff
@@ -1,259 +1,263 @@
       },
       "elasticsearch-metrics": {
         "image": "tigera/elasticsearch-metrics",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "packetcapture": {
         "image": "tigera/packetcapture",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "prometheus": {
         "image": "tigera/prometheus",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "coreos-prometheus": {
-        "version": "v2.42.0"
+        "version": "v2.43.1"
       },
       "prometheus-operator": {
         "image": "tigera/prometheus-operator",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "prometheus-config-reloader": {
         "image": "tigera/prometheus-config-reloader",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "tigera-prometheus-service": {
         "image": "tigera/prometheus-service",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "es-gateway": {
         "image": "tigera/es-gateway",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "deep-packet-inspection": {
         "image": "tigera/deep-packet-inspection",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "eck-elasticsearch-operator": {
         "version": "2.6.1"
       },
       "elasticsearch-operator": {
         "image": "tigera/eck-operator",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "coreos-alertmanager": {
-        "version": "v0.25.0"
+        "version": "v0.25.1"
       },
       "alertmanager": {
         "image": "tigera/alertmanager",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "envoy": {
         "image": "tigera/envoy",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "envoy-init": {
         "image": "tigera/envoy-init",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "windows": {
         "image": "tigera/calico-windows",
-        "version": "master"
+        "version": "v.3.18.0"
       },
       "windows-upgrade": {
         "image": "tigera/calico-windows-upgrade",
-        "version": "master"
+        "version": "v.3.18.0"
+      },
+      "policy-recommendation": {
+        "image": "tigera/policy-recommendation",
+        "version": "v.3.18.0"
       },
       "flexvol": {
-        "image": "calico/pod2daemon-flexvol",
-        "version": "master",
+        "image": "tigera/pod2daemon-flexvol",
+        "version": "v.3.18.0",
         "registry": "quay.io"
       },
       "csi-driver": {
-        "image": "calico/csi",
-        "version": "master",
-        "registry": "quay.io"
-      },
-      "csi-node-driver-registrar": {
-        "image": "calico/node-driver-registrar",
-        "version": "master",
+        "image": "tigera/csi",
+        "version": "v.3.18.0",
         "registry": "quay.io"
       }
+    },
```

Comment:
Indentation seems wrong.

Reply:
Yes. @bmckercher123 you'll need to move "csi-node-driver-registrar" back in line with the others. Should sort out your build issues.(At least it did for me locally.)

Reply:
Fixed.